service-tests/microbadger.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'microbadger', title: 'MicroBadger' })
+module.exports = t;
+
+t.create('image size without a specified tag')
+  .get('/image-size/_/hello-world.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'image size',
+    value: Joi.string().regex(/^[0-9]+[kMGTPEZY]B$/)
+  }));
+
+t.create('image size with a specified tag')
+  .get('/image-size/_/httpd/alpine.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: 'image size',
+    value: Joi.string().regex(/^[0-9]+[kMGTPEZY]B$/)
```

Comment:
I think the convention for this project is that when writing a service test for a badge that uses `metric()`, the test should make a call that should generate a non-zero value and check the result with a regex that ensures the badge value is non-zero: `/^[1-9][0-9]*[kMGTPEZY]?$/`. See https://github.com/badges/shields/blob/master/service-tests/helpers/validators.js#L38-L43

Reply:
We also have [isFileSize](https://github.com/badges/shields/blob/master/service-tests/helpers/validators.js#L46), though probably it should be modified to reject 0 as well!